e2e/scenario_win_test.go

```diff
@@ -25,6 +25,7 @@ func Test_Windows2019Containerd(t *testing.T) {
 				ValidateFileHasContent(ctx, s, "/k/kubeletstart.ps1", "--container-runtime=remote")
 				ValidateWindowsProcessHasCliArguments(ctx, s, "kubelet.exe", []string{"--rotate-certificates=true", "--client-ca-file=c:\\k\\ca.crt"})
 				ValidateCiliumIsNotRunningWindows(ctx, s)
+				ValidateFileHasContent(ctx, s, "/k/test.txt", "this is a test file")
```

Comment:
These are currently here to check the e2e tests are being run with the right VHD version. They will be removed before I merge the PR.